mindjunkies/dashboard/forms.py

```
@@ -0,0 +1,9 @@
+from django import forms
+from .models import TeacherVerification, Certificate
```

Ruff (0.8.2)

`.models.Certificate` imported but unused

Remove unused import: `.models.Certificate`

(F401)